plugins/pay.c

```diff
@@ -474,6 +488,15 @@ static struct command_result *listsendpays_done(struct command *cmd,
 		key.payment_hash = &payment_hash;
 		key.groupid = groupid;
 
+		created_indextok = json_get_member(buf, t, "created_index");
+		updated_indextok = json_get_member(buf, t, "updated_index");
+		assert(created_indextok != NULL);
+		json_to_u64(buf, created_indextok, &created_index);
+		if (updated_indextok != NULL)
+			json_to_u64(buf, updated_indextok, &updated_index);
+		else
+			updated_index = UINT64_MAX;
```

Comment:
Shouldn't this be 0, so it's ignored above?

Reply:
eh yes probably